src/main/java/uk/nhs/digital/nhsconnect/lab/results/model/edifact/SpecimenCollectionDateTime.java

```diff
@@ -45,10 +45,9 @@ public static SpecimenCollectionDateTime fromString(final String edifactString)
         final SpecimenCollectionDateTimeBuilder collectionDateTimeBuilder = SpecimenCollectionDateTime.builder();
 
         if (isBlank(collectionDateTime) || isBlank(format)) {
-            throw new IllegalArgumentException("Can't create " + SpecimenCollectionDateTime.class.getSimpleName()
-                + " from " + edifactString + ". Both the collection date time and the time format should be presented");
+            throw new IllegalArgumentException("Can't create SpecimenCollectionDateTime from " + edifactString
+                + " because of missing date-time and/or format definition");
```

Comment:
We can still use the refactor-safe approach - I just wanted to make my suggestion code clearer :)
```suggestion
            throw new IllegalArgumentException("Can't create " + SpecimenCollectionDateTime.class.getSimpleName() 
                + " from " + edifactString + " because of missing date-time and/or format definition");
```